install/page_dbconnection.php

```diff
@@ -46,8 +46,8 @@
 
 $error = '';
 if ($_SERVER['REQUEST_METHOD'] == 'POST' && !empty( $vars['DB_HOST'] ) && !empty( $vars['DB_USER'] )) {
-	$func_connect = empty( $vars['DB_PCONNECT'] ) ? "mysql_connect" : "mysql_pconnect";
-	if (! ( $link = @$func_connect( $vars['DB_HOST'], $vars['DB_USER'], $vars['DB_PASS'], true ) )) {
+    $hostname = empty( $vars['DB_PCONNECT'] ) ? $vars['DB_HOST'] : "p:".$vars['DB_HOST'];
+    if (! ( $link = @mysqli_connect( $hostname, $vars['DB_USER'], $vars['DB_PASS'], true ) )) {
```

Comment:
DB name needs to be added here, fourth param is the db name, I think the 'true' is leftover from the old syntax...probably don't need that at all? See what the other params are and if anything further past db name makes sense to include.